performance-tuning/99-smp-packet-steering-minim

```diff
@@ -0,0 +1,21 @@
+#!/bin/sh
+[ "$ACTION" = add ] || exit
+
+NPROCS="$(grep -c "^processor.*:" /proc/cpuinfo)"
+[ "$NPROCS" -gt 1 ] || exit
+
+PROC_MASK="$(( (1 << $NPROCS) - 1 ))"
```

Comment:
used?

Reply:
No, I left it in as the other scripts have it.